cli/cmd/devtool/start.go

```diff
@@ -269,7 +288,13 @@ func (s cloud) start(ctx context.Context, ch *cmdutil.Helper, verbose, reset, re
 	logInfo.Printf("State directory is %q\n", stateDirectory())
 
 	if services.deps {
-		g.Go(func() error { return s.runDeps(ctx, verbose, preset) })
+		var profiles []string
+		if services.minimal {
+			profiles = []string{"minimal"}
+		} else {
+			profiles = []string{preset} // e.g., "cloud", "e2e"
+		}
```

Comment:
Actually, why not set minimal as a preset instead of a separate flag? It feels overly complicated to have all of `preset`, `--minimal`, `--only` and `--except`.